### ISSUE TITLE
TypeError: StableDiffusionControlNetPipeline.prepare_image() missing 1 required positional argument:  'do_classifier_free_guidance'

### ISSUE THREAD

**Issue by Amazingldl** [mikonvergence/ControlNetInpaint#8]
```
pipe.to('cuda')

# generate image
generator = torch.manual_seed(0)
new_image = pipe(
    text_prompt,
    num_inference_steps=20,
    generator=generator,
    image=image,
    control_image=canny_image,
    controlnet_conditioning_scale = 0.5,
    mask_image=mask_image
).images[0]

new_image.save('output/canny_result.png')
```
Thanks for your great work, by running the above code in notebook, I get some issues:
```
╭─────────────────────────────── Traceback (most recent call last) ────────────────────────────────╮
│ in <module>:5                                                                                    │
│                                                                                                  │
│    2                                                                                             │
│    3 # generate image                                                                            │
│    4 generator = torch.manual_seed(0)                                                            │
│ ❱  5 new_image = pipe(                                                                           │
│    6 │   text_prompt,                                                                            │
│    7 │   num_inference_steps=20,                                                                 │
│    8 │   generator=generator,                                                                    │
│                                                                                                  │
│ d:\App\miniconda\envs\aigc\lib\site-packages\torch\autograd\grad_mode.py:27 in decorate_context  │
│                                                                                                  │
│    24 │   │   @functools.wraps(func)                                                             │
│    25 │   │   def decorate_context(*args, **kwargs):                                             │
│    26 │   │   │   with self.clone():                                                             │
│ ❱  27 │   │   │   │   return func(*args, **kwargs)                                               │
│    28 │   │   return cast(F, decorate_context)                                                   │
│    29 │                                                                                          │
│    30 │   def _wrap_generator(self, func):                                                       │
│                                                                                                  │
│ c:\Users\Arthur\Downloads\ControlNetInpaint-main\ControlNetInpaint-main\src\pipeline_stable_diff │
│ usion_controlnet_inpaint.py:394 in __call__                                                      │
│                                                                                                  │
│   391 │   │   )                                                                                  │
│   392 │   │                                                                                      │
│   393 │   │   # 4. Prepare image                                                                 │
│ ❱ 394 │   │   control_image = self.prepare_image(                                                │
│   395 │   │   │   control_image,                                                                 │
│   396 │   │   │   width,                                                                         │
│   397 │   │   │   height,                                                                        │
╰──────────────────────────────────────────────────────────────────────────────────────────────────╯
TypeError: StableDiffusionControlNetPipeline.prepare_image() missing 1 required positional argument: 
'do_classifier_free_guidance'
```

**Comment by mikonvergence** [comment]
Can you confirm what `diffusers` version you're using (`diffusers.__ver__`)?

The latest `0.15.0` update of `diffusers` has some inconsistencies in the interface, as mentioned in #7. For now, you can fix this by using `diffusers==0.14.0`. As mentioned in the other thread, I will wait a week or so to see if the interface gets fixed on the side of diffusers.

If it does stay that way, I will make changes to this tool to make it work with all versions of diffusers.

**Comment by Amazingldl** [comment]
You are so kind, thank you for your quick reply!
I will install `diffusers==0.14.0` instead of 0.15.0.

> Can you confirm what `diffusers` version you're using (`diffusers.__ver__`)?
> 
> The latest `0.15.0` update of `diffusers` has some inconsistencies in the interface, as mentioned in #7. For now, you can fix this by using `diffusers==0.14.0`. As mentioned in the other thread, I will wait a week or so to see if the interface gets fixed on the side of diffusers.
> 
> If it does stay that way, I will make changes to this tool to make it work with all versions of diffusers.
